features/step_definitions/viewing_browse_steps.rb

```diff
@@ -77,3 +82,43 @@
 Then(/^I should see the topics linked under Detailed guidance$/) do
   assert page.has_content?('A linked topic')
 end
+
+def top_level_browse_pages
+  [
+    { title: 'Crime and justice', base_path: '/browse/crime-and-justice' },
+    { title: 'Benefits', base_path: '/browse/benefits' },
+  ]
+end
+
+def add_browse_pages
+  content_store_has_item '/browse', { links: {
+    top_level_browse_pages: top_level_browse_pages
+  }}
+end
+
+def add_first_level_browse_pages(second_level_browse_pages, order_type)
+  content_store_has_item('/browse/crime-and-justice', {
+    base_path: '/browse/crime-and-justice',
+    links: {
+      top_level_browse_pages: top_level_browse_pages,
+      second_level_browse_pages: second_level_browse_pages,
+    },
+    details: {
+      second_level_ordering: order_type,
+      ordered_second_level_browse_pages: second_level_browse_pages.map { |page| page[:content_id] }
```

Comment:
Can `ordered_second_level_browse_page` ever be a subset of the the full list instead of the full thing? If so we should have a test for it. I would expect non curated pages to be ignored.